phpstan-baseline.neon

```
@@ -39,3 +39,13 @@ parameters:
 			message: "#^Call to an undefined method Sonata\\\\AdminBundle\\\\Datagrid\\\\ProxyQueryInterface\\:\\:.+\\(\\)\\.$#"
 			count: 13
 			path: src/Filter/TranslationFieldFilter.php
+
+		-
+			message: "#^Parameter \\#2 \\$showCountryFlagsOrTemplating of class Sonata\\\\TranslationBundle\\\\Block\\\\LocaleSwitcherBlockService constructor expects bool\\|Symfony\\\\Bundle\\\\FrameworkBundle\\\\Templating\\\\EngineInterface\\|null, stdClass given\\.$#"
+			count: 1
+			path: tests/Block/LocaleSwitcherBlockServiceTest.php
+
+		-
+			message: "#^Method Sonata\\\\TranslationBundle\\\\Tests\\\\Fixtures\\\\Model\\\\Knplabs\\\\TranslatableEntity\\:\\:getLocale\\(\\) should return string but returns Knp\\\\DoctrineBehaviors\\\\Model\\\\Translatable\\\\Returns\\.$#"
```

https://github.com/KnpLabs/DoctrineBehaviors/blob/072078422bcada395906ec96b192c6b30a93bebb/src/Model/Translatable/TranslatableMethods.php#L149

The phpdoc is wrong, this will be removed with [knplabs/doctrine-behaviors 2](https://github.com/sonata-project/SonataTranslationBundle/pull/416)

---

Shouldnt we add a comment for every ignored error in order to know when they will be solve ; like we did for SonataAdmin